universal-application-tool-0.0.1/app/views/style/BaseStyles.java

```diff
@@ -11,11 +11,11 @@
   // CiviForm color classes
   /////////////////////////////////////////////////////////////////////////////////////////////////
 
-  public static final String BG_CIVIFORM_WHITE = "bg-civiformWhite";
+  public static final String BG_CIVIFORM_WHITE = "bg-civiform-white";
 
-  public static final String BG_SEATTLE_BLUE = "bg-seattleBlue";
-  public static final String TEXT_SEATTLE_BLUE = "text-seattleBlue";
-  public static final String BORDER_SEATTLE_BLUE = "border-seattleBlue";
+  public static final String BG_SEATTLE_BLUE = "bg-seattle-blue";
```

Comment:
(blocking) Did you actually verify that this works? I tried using 'seattle-blue' in the tailwind.config.js before, but the resulting tailwind class string was still actually "bg-seattleBlue".

Reply:
Yup. The following styles are applied:

```
.bg-seattle-blue {
    --tw-bg-opacity:  1;
    background-color: rgba(17, 63, 159, var(--tw-bg-opacity));
}
```

Reply:
Oh great! I wonder why I had trouble getting that working before. Maybe I just hading properly restarted my dev server or something. This is defs an improvement - thank you!